data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/ArithmeticExpressionEvaluatorIT.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.JacksonEvent;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+import java.util.Random;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.isA;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.sameInstance;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.not;
+import org.apache.commons.lang3.RandomStringUtils;
+
+class ArithmeticExpressionEvaluatorIT {
```

Comment:
Since there is no `ArithmeticExpressionEvaluator` class, is this really an IT for `GenericExpressionEvaluator`?

Reply:
Another approach that is fairly common in the Java world is to split tests using an underscore. This is testing the `GenericExpressionEvaluator` for arithmetic and you have another testing `GenericExpressionEvaluator` for strings.

Thus:

* `GenericExpressionEvaluator_ArithmeticIT`
* `GenericExpressionEvaluator_StringIT`

This has the benefit of putting them next to each other in file tree UIs.

I'm fine keeping what you have or using this pattern.

Reply:
Will rename to GenericExpressionEvaluator_ArithmeticIT and GenericExpressionEvaluator_ArithmeticIT